__tests__/sources.ts

```diff
@@ -186,6 +186,37 @@ query SourceByFeed($data: String!) {
   });
 });
 
+describe('query source current member', () => {
```

Comment:
Can you please add a test of an anonymous user and user which is not a source member?

Reply:
It should fail right? as it's trying to query role which is required on a member?

Reply:
I'd say return null. We wouldn't want the whole query to fail in this case.
It's a valid use case for public squads for example

Reply:
Yep agreed, modified the query and test cases now.